views/privacy-notice.erb

```diff
@@ -0,0 +1,159 @@
+
+			<h2 class="heading-small">Why we need data from tenants</h2>
+			<p>We use this information to:</p>
+			<ul class="list list-bullet">
+				<li>create and manage user accounts on GOV.UK PaaS</li>
+				<li>identify you while you use it </li>
+				<li>keep the platform secure from unauthorised access</li>
+			</ul>
+
+			<h2 class="heading-small">What we do with tenants’ data</h2>
+			<p>We’ll store the data you provide to:</p>
+			<ul class="list list-bullet">
+				<li>make your user account function correctly</li>
+				<li>manage your user account</li>
+				<li>send you updates and notices </li>
+			</ul>
+			<p>We will not:</p>
+			<ul class="list list-bullet">
+				<li>sell or rent your data to 3rd parties</li>
+				<li>share your data with 3rd parties for marketing purposes</li>
+			</ul>
+			<p>We will share your data if we are required to do so by law – for example, by court order, or to prevent fraud or other crime.</p>
+
+
+			<h2 class="heading-small">How long we keep data from tenants</h2>
+			<p>We will only retain your personal data for as long as:</p>
+			<ul class="list list-bullet">
+				<li>the law requires us to</li>
+				<li>we need to provide this service </li>
+			</ul>
+			<p>In general, this means that we will only hold your personal data for a minimum of 1 year and a maximum of 2 years.</p>
+			<p>We will delete your personal data when you ask us to remove your user account. It might take up to 35 days for your data to be completely cleared from our logs and back-ups.</p>
+
+			<h2 class="heading-medium">What data we need from end users</h2>
+			<p>The personal data we collect from end users will include:</p>
+			<ul class="list list-bullet">
+				<li>their IP address</li>
+				<li>any cookies that are set by your service</li>
+			</ul>
+			<p>We need this information to provide a service that is in the public interest, as the apps we host on GOV.UK PaaS are used by the public to access government services. </p>
+
+			<h2 class="heading-small">Why we need your end users’ data</h2>
+			<p>We need this information to make sure the platform can meet the demand created by end users who access the services hosted on it.</p>
+
+			<h2 class="heading-small">What we do with your end users’ data</h2>
+			<p>We use the data provided to:</p>
+			<ul class="list list-bullet">
+				<li>ensure the security of your service if any cookie you set has that purpose </li>
+				<li>monitor the load placed on the platform</li>
+				<li>monitor the security of the GOV.UK PaaS </li>
+			</ul>
+			<p>We will not:</p>
+			<ul class="list list-bullet">
+				<li>sell or rent your end users’ data to 3rd parties</li>
+				<li>share your end users’ data with 3rd parties for marketing purposes </li>
+			</ul>
+			<p>We will share your end users’ data if we are required to do so by law – for example, by court order, or to prevent fraud or other crime.</p>
+
+			<h2 class="heading-small">How long we keep your end users’ data</h2>
```

Comment:
For any `heading-small` class, we should probably use `h3` instead. Not a blocker.